### DISCUSSION TITLE
Separating components of additive/multiplicative kernels

### DISCUSSION THREAD

**Discussion by fonnesbeck** [cornellius-gp/gpytorch#1549] [Q&A]
Is there an easy way of breaking out the components of an additive or multiplicative kernel after fitting them in GPyTorch? I'm aiming to plot them separately to give an idea of the contribution of each component to the overall model.

**Comment by jacobrgardner** [comment]
Both kernels just use a module list to store their components: https://github.com/cornellius-gp/gpytorch/blob/2e55ec6cf6bd70dc96db76d88689dd743fd42ade/gpytorch/kernels/kernel.py#L460
